Sources/Fullscreen/Consent/Details/ConsentActionView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Copyright © 2018 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public class ConsentActionView: UIView {
+
+    // MARK: - Private properties
+
+    private lazy var textLabel: Label = {
+        let label = Label(style: .body)
+        label.textColor = .stone
```

Comment:
Was wondering that we should place the `translatesAutoresizingMaskIntoConstraints` call just below the instantiation `let label = Label(style: .body)` but maybe is too much nitpicking?

Reply:
Sure, I can do that. I've just put it at the end out of habit